eng/testing/performance/performance-setup.sh

```diff
@@ -249,12 +259,18 @@ common_setup_arguments="--channel main --queue $queue --build-number $build_numb
 setup_arguments="--repository https://github.com/$repository --branch $branch --get-perf-hash --commit-sha $commit_sha $common_setup_arguments"
 
 if [[ "$run_from_perf_repo" = true ]]; then
+    echo "Running from perf repo"
     payload_directory=
     workitem_directory=$source_directory
     performance_directory=$workitem_directory
     setup_arguments="--perf-hash $commit_sha $common_setup_arguments"
 else
-    git clone --branch main --depth 1 --quiet https://github.com/dotnet/performance $performance_directory
+    echo "Not running from perf repo"
```

Comment:
line 263-269 should be revert back to the original code to clone main branch once PR https://github.com/dotnet/performance/pull/1902 is merged